akka-actor-tests/src/test/scala/akka/util/BoundedBlockingQueueSpec.scala

```diff
@@ -115,6 +114,7 @@ class BoundedBlockingQueueSpec
       val f = Future(queue.put("b"))
 
       after(10 milliseconds) {
+        f.isCompleted should be(false)
         queue.take()
       }
 
```

Comment:
no reason to have such tight timeout 100 milliseconds on next line, see https://github.com/akka/akka/blob/master/CONTRIBUTING.md#preferred-ways-to-use-timeouts-in-tests

Reply:
in the light of that, are there other assumptions about time in this test that will not work on slow machines with hiccups?

Reply:
The `Await.result` might indeed trigger a failure if it times out too early.

Some other assumptions:
- when the code in `after(..)` runs, I assume the future above to be started and blocked. A slow thread might cause the order of `take` and `put` to be reversed but this should not cause the test to fail when it shouldn't.
- calls to `mustBlockFor` also have this 100 ms timeout. I can raise this to the 3 seconds as specified in the guidelines, but this would definitely make 7 tests quite a bit slower.

In both cases, chances are that a slow thread will cause a regression to remain undetected, but it will never trigger a false positive.

For now I'll increase the `Await.result` timespans. If there's anything else, let me know.